collectors/python.d.plugin/nvidia_smi/nvidia_smi.chart.py

```diff
@@ -474,6 +481,14 @@ def data(self):
             'power_draw': self.power_draw(),
         }
 
+        if self.pci_bw_max() is None or self.pci_bw_max() == 0:
```

Comment:
```suggestion
        pci_bw_max = self.pci_bw_max()
        if not pci_bw_max:
            ...
```

nit-pick:
  - call `self.pci_bw_max()` once, later use `pci_bw_max`.
  - use `not pci_bw_max` instead `is None or == 0` (does the same job).